app/dashboard/helpers.py

```diff
@@ -542,11 +545,33 @@ def process_bounty_details(bounty_details):
             new_bounty = create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id)
 
             if new_bounty:
+                create_new_feedback(new_bounty, bounty_review)
                 return (did_change, latest_old_bounty, new_bounty)
             return (did_change, latest_old_bounty, latest_old_bounty)
     except SemaphoreExists:
         return (did_change, latest_old_bounty, latest_old_bounty)
 
+def create_new_feedback(new_bounty, feedback_dict):
```

Comment:
ever considered putting this logic into model ? models looks anemic and you create 'create' method in helpers while it can belong to Feedback itself
```
@classmethod
worker_feedback(cls,bounty,...)
  return cls()

@classmethod
other_worker_feedback(cls,bounty,...) 
  return cls()
```

Reply:
yes, considered that already. The main thing was to get it working, since we are having a IPFS bridge here and it wasn't working very well. especially because we do not simply provide the IPFS later on to the create functions, but actually parse them and return a DIFFERENT json structure than the one we pass into IPFS. Hence those methods are still from the "trying to get this to work" era.